nearlib/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "nearlib",
   "description": "Javascript library to interact with near blockchain",
-  "version": "0.5.4",
+  "version": "0.5.5",
```

Comment:
@evgenykuzyakov isn't this change breaking API? Means this should be version 0.6.0, see http://semver.org

Reply:
It's non-breaking change, but it does require certain version of nearcore that has proto changes.

Reply:
But it's not backward compatible, so bumping to 0.6.0.
Not going to publish it until we add the new signer